scripts/ensure-pinned-deps.ts

```diff
@@ -14,14 +14,28 @@
  * limitations under the License.
  */
 
+import {readdirSync, readFileSync} from 'fs';
+import {join} from 'path';
 import packageJson from '../package.json';
 
+const LOCAL_PACKAGE_NAMES = [];
```

Comment:
This should be a `Set`

Reply:
Sets are generally slower for a small number of objects. Even for scanning operations.